src/git/NOTES.md

```diff
@@ -2,6 +2,6 @@
 
 ## OS Support
 
-This Feature should work on recent versions of Debian/Ubuntu-based distributions with the `apt` package manager installed.
+This Feature should work on recent versions of Debian/Ubuntu, RedHat Enterprise Linux, Fedora, Alma, and RockyLinux distributions with the `apt`, `yum`, or `dnf` package manager installed.
```

Comment:
🎉 🎉

Reply:
Should this mention about `microdnf` package manager as well?

Reply:
Done. commit with `microdnf` included coming soon.